src/soda-js.coffee

```diff
@@ -87,7 +87,14 @@ expr =
   lt:  (column, literal) -> "#{column} < #{handleLiteral(literal)}"
   lte: (column, literal) -> "#{column} <= #{handleLiteral(literal)}"
   eq:  (column, literal) -> "#{column} = #{handleLiteral(literal)}"
-
+  
+# serialize object to querystring
+serialize = (obj) ->
+  str = []
+  for p of obj
+    if obj.hasOwnProperty(p)
```

Comment:
Also, it's slightly more idiomatic to use the key/value notation when iterating over objects (scroll down a little bit in the [Loops](http://coffeescript.org/#loops) section of CoffeeScript docs to learn more):

```
for own key, val of obj
  str.push encodeURIComponent(key) + '=' + encodeURIComponent(val)
```